shipyard/README.md

```diff
@@ -0,0 +1,56 @@
+# Shipyard
+
+## Overview
+
+Shipyard unblocks DevOps teams from bulky, outdated production pipelines. We resolve the pre-production or staging bottleneck with our agile on-demand ephemeral (testing or preview) environments.
+
+The Datadog Shipyard integration automatically passes your [Shipyard][1] ephemeral environment logs to your Datadog dashboard. This is valuable for using Datadog's [log analysis and correlation][8] functions and helping your team with [shift-left testing][9].
+
+For every application you want the logs for, Shipyard sends the runtime logs of your application to Datadog. 
```

Comment:
Please include what kinds of logs will be sent in, and the value that these logs will bring for a Datadog/Shipyard customer - i.e. why they want these logs, and how they can use them in Datadog. 

In this case, it links to various articles which could be a lot of redirecting for customers trying to understand the value of the integration.

Reply:
Hi @emarsha94, 

So the type of logs we send to Datadog is the runtime logs of an application. What are you looking for in clarification on this sentence?

Reply:
That's perfect! Can you include that and the value of sending runtime logs to Datadog? This section is to help mutual customers or new customers understand what the integration does and the value of the integration.